examples/basic_usage/main.tf

```diff
@@ -13,11 +17,12 @@ provider "aws" {
 module "example" {
   source = "../../"
   providers = {
-    aws = aws
+    aws       = aws
+    aws.users = aws.users
   }
 
-  ami_owner_account_id  = var.ami_owner_account_id
-  aws_availability_zone = var.aws_availability_zone
-  aws_region            = var.aws_region
-  subnet_id             = aws_subnet.example.id
+  account_ids                 = var.account_ids
+  role_name                   = "ReadTerraformStateReadRoleTFModuleTerraformState"
+  terraform_state_bucket_name = "cisa-cool-terraform-state"
```

Comment:
```suggestion
  terraform_state_bucket_name = "example-terraform-state-bucket"
```

Reply:
See comment above.  I'm going to keep the bucket name as is, unless you strongly disagree with that plan.